libs/sr-db/src/ICharacter.ts

```diff
@@ -0,0 +1,21 @@
+import type {
+  AscensionKey,
+  BonusAbilityKey,
+  CharacterKey,
+  StatBoostKey,
+} from '@genshin-optimizer/sr-consts'
+
+export interface ICharacter {
+  key: CharacterKey
+  level: number
+  eidolon: number
+  ascension: AscensionKey
+  traces: {
```

Comment:
probably easier to keep this as flat as possible, since this traces object doesnt really do much